src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,15 +6,28 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandJoystick;
+import frc.robot.constants.PhoenixDriveConstants;
+import frc.robot.subsystems.drive.PhoenixDrive;
+import frc.robot.subsystems.drive.commands.DriveWithJoysticks;
 
 public class RobotContainer {
-  public RobotContainer() {
-    configureBindings();
-  }
+    PhoenixDrive drive = PhoenixDriveConstants.DriveTrain;
+    Telemetry logger = new Telemetry(6);
+    CommandJoystick leftJoystick = new CommandJoystick(0);
+    CommandJoystick rightJoystick = new CommandJoystick(1);
 
-  private void configureBindings() {}
+    public RobotContainer() {
+        configureBindings();
+    }
 
-  public Command getAutonomousCommand() {
-    return Commands.print("No autonomous command configured");
-  }
+    private void configureBindings() {
+        System.out.println("add binding");
```

Comment:
we can remove the prints since everything is working now with the setControl update

Reply:
done